shadows/framework/src/main/java/org/robolectric/shadows/ShadowLegacyChoreographer.java

```diff
@@ -5,13 +5,13 @@
 import android.os.SystemClock;
 import android.view.Choreographer;
 import android.view.Choreographer.FrameCallback;
+import java.time.Duration;
```

Comment:
`Duration` is available since API 26, or with JDK desuguaring.
Is it fine to use it here?